drop-in/src/main/java/com/adyen/checkout/dropin/SessionDropInResultContractParams.kt

```diff
@@ -9,7 +9,6 @@
 package com.adyen.checkout.dropin
 
 import androidx.annotation.RestrictTo
-import com.adyen.checkout.dropin.service.SessionDropInService
 import com.adyen.checkout.sessions.core.CheckoutSession
 
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
Should also be moved to internal package

Reply:
And I think we can make it an `internal class`

Reply:
This class is used in `DropIn.registerForDropInResult()` as a result so I guess it can be `internal` also use as a parameter for `DropIn.startPayment()`

Reply:
Oh right, it's the generic type of the `ActivityResultLauncher`